Running tests in an empty project failing

When running tests (npm test) in a project without any tests yet, we receive linting errors about temp files, but no linting errors about our own files.

```
/path/to/client/.tmp/mocha-webpack/917d4d4eeb2f8fd0100e585ed757ded9-entry.js
  2:40  warning  Strings must use singlequote                   quotes
  7:1   error    Trailing spaces not allowed                    no-trailing-spaces
  7:2   error    Newline required at end of file but not found  eol-last
```

--- a/etc/webpack.test.js
+++ b/etc/webpack.test.js
@@ -9,11 +9,11 @@ module.exports = helpers.extend(
       preLoaders: [{
         test: /\.jsx?$/,
         loader: 'eslint',
-        exclude: /node_modules/
+        exclude: [/node_modules/, /.tmp/] 
       }, {
         test: /\.css$/,
         loader: 'stylelint',
-        exclude: /node_modules/
+        exclude: [/node_modules/, /.tmp/]
       }],
       loaders: [{
         test: /\.test\.jsx?$/,

